sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,20 +1,35 @@
 # Release History
 
-## 1.9.0-beta.2 (Unreleased)
+## 1.10.0-beta.1 (2023-05-04)
 
 ### Features Added
 
 - Added `Azure::Core::Uuid::AsArray()` and `Azure::Core::Uuid::CreateFromArray()` to enable reading or writing from an existing UUID.
 This is useful when the UUID was generated outside the Azure SDK, or needs to be used from a component outside the Azure SDK.
 
-### Breaking Changes
-
 ### Bugs Fixed
 
 - Fixed the UUID generation so the variant is RFC 4122 conforming.
```

Comment:
Unless special circumstances dictate otherwise (say OOB release), this should be GA'd (and all future bug fixes in main).

If we have to select things to be kept out of the monthly GA release, they should primarily be features and new API surface that are not ready to be GA'd and need beta releases for stability. Everything else should be GA-able.